listenbrainz/webserver/views/stats_api.py

```diff
@@ -86,7 +92,8 @@ def get_artist(user_name):
         return '', 204
 
     if count is None:
-        count = stats['artist']['count']
+        count = DEFAULT_ITEMS_PER_GET
```

Comment:
instead of doing this check here, we could just do `count = _get_non_negative_param('count', default=DEFAULT_ITEMS_PER_GET) above on line 84, where we first retrieve count,